contrib/scripts/conventional-commits

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
+
+# list of Conventional Commits types
+cc_types=("feat" "fix")
+default_types=("build" "chore" "ci" "docs" "${cc_types[@]}" "perf" "refactor" "revert" "style" "test")
+types=( "${cc_types[@]}" )
+
+if [ $# -eq 1 ]; then
+    types=( "${default_types[@]}" )
+else
+    while [ $# -gt 1 ]; do
+        types+=( "$1" )
+        shift
+    done
+fi
+
+msg_file="$1"
+
+r_types="($(IFS='|'; echo "${types[*]}"))"
+r_scope="(\([[:alnum:] \/-]+\))?"
+r_delim='!?:'
+r_subject=" [[:print:]].+"
+pattern="^$r_types$r_scope$r_delim$r_subject$"
+
+if grep -Eq "$pattern" "$msg_file"; then
```

Comment:
is it possible to omit the check if needed urgently?

Reply:
Kinda.

This is a client-side check, if you don't explicitly install the `pre-commit` it will not run.

This doesn't reject anything server-side.

If you're smart enough to install it, you can probably put a colon in the middle of the tag and make it push...